### DISCUSSION TITLE
Reading My Own Blog Posts (no bots!) - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#16] [General]
# Reading My Own Blog Posts (no bots!) - Alan Pope's blog

Download audio I had some fun when I blogged about using a bot to read my blog post. While fun, it wasn’t a particularly pleasant way to consume blog content. The audio is still a bit robotic, with little care for timing, ephasis and stress on words. So in my next blog post, in which I detailed how to setup Mimic 3, I actually read the blog post out loud, recorded that and attached it as an MP3.

https://popey.com/blog/2022/10/reading-my-own-blog-posts/

**Comment by gripfastistech** [comment]
:thumbsup:
